src/components/Dropdown.vue

```diff
@@ -50,13 +63,24 @@ export default {
       type: Boolean,
       default: false,
     },
+    defaultValue: {
```

Comment:
using this where?

Reply:
Oh, I was testing something. Forgot to remove it.